### PR TITLE
Merge master into features

### PR DESCRIPTION
**Pull request by RonnyPfannschmidt**
.

### CODE REVIEW

**Comment by nicoddemus** [comment]
@RonnyPfannschmidt I think there was some problem with the merge

File: AUTHORS [hunk 1]
```diff
@@ -82,12 +89,25 @@ Raphael Pierzina
...
+Trevor Bekolay
+Wouter van Ackooy
+David Díaz-Barquero
+Eric Hunsberger
+Simon Gomizelj
+Russel Winder
+Ben Webb
+Alexei Kozlenok
+Cal Leeming
+Feng Ma
```
Comment (The-Compiler) [inline AUTHORS]: This seems to duplicate people which are already added above.

Comment (nicoddemus) [inline AUTHORS]: Ooops, that's my fault, didn't check for that when I pushed this. :grimacing:

**Comment by RonnyPfannschmidt** [comment]
closing due to merge mistakes

**Pull request closed without merging**